crates/miden-objects/src/account/storage/slot/slot_name.rs

```diff
@@ -37,7 +35,8 @@ use crate::utils::serde::{ByteWriter, Deserializable, DeserializationError, Seri
 /// - Each component must not start with an underscore.
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub struct StorageSlotName {
-    name: Cow<'static, str>,
+    name: String,
```

Comment:
Does keeping it `Cow<'static, str>` complicate things? Mostly curious if `String` is more efficient than `Cow<'static, str>`.

Reply:
It just removes one layer of indirection, e.g. `Cow::Owned` is just a `String` anyway and we never use the `Cow::Borrowed` version now, so seems simpler to do that. In terms of efficiency, it's probably pretty much the same.